iac-extensions/docker/src/test/resources/checks/WorkdirAbsolutPathCheck/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM scratch
+
+WORKDIR /absolute/path/unix
+WORKDIR "/absolute/path/unix"
+
```

Comment:
`WORKDIR ""` would be non compliant, is this expected?

Reply:
Indeed, even if it's a bit odd, doesn't reflect well if something like that appear on SonarLint. I will handle this use case.